File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.c
#include <stdio.h>

int main()
{
    int n, m, numArr1[101][101], numArr2[101][101];

    scanf("%d %d", &n, &m);

    for (int i = 0; i < n; i++) {
        for (int j = 0; j < m; j++) {
            scanf("%d", &numArr1[i][j]);
        }
    }
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < m; j++) {
            scanf("%d", &numArr2[i][j]);
        }
    }
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < m; j++) {
            printf("%d ", numArr1[i][j] + numArr2[i][j]);
        }
        printf("\n");
    }
    return 0;
}

File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.py
import sys
input = sys.stdin.readline
n, m = map(int, input().split())
numArr1 = []
numArr2 = []
for i in range(n):
    numArr1.append(list(map(int, input().split())))
for i in range(n):
    numArr2.append(list(map(int, input().split())))
for i in range(n):
    for j in range(m):
        print(numArr1[i][j] + numArr2[i][j], end = " ")
    print()